gramps/gen/lib/eventtype.py

```diff
@@ -280,6 +284,13 @@ def is_death(self):
         """
         return self.value == self.DEATH
 
+    def is_stillborn(self):
```

Comment:
Is this method needed?

Reply:
Removed the def section and it still worked so obviously was not needed.

Fixed in [15a0085](https://github.com/gramps-project/gramps/pull/1148/commits/15a0085b6316bb277400a9133eabf32b47be2558)